diff --git a/ddsmt/progress.py b/ddsmt/progress.py
index 2c36c06..90fe8bf 100644
--- a/ddsmt/progress.py
+++ b/ddsmt/progress.py
@@ -19,7 +19,6 @@
 # along with ddSMT.  If not, see <https://www.gnu.org/licenses/>.
 
 import logging
-import progressbar
 import sys
 
 __BAR = None
@@ -36,11 +35,6 @@ def start(max):
         return
     if logging.getLogger().level > logging.INFO:
         return
-    widgets = [progressbar.Bar(), ' ', progressbar.Counter(), ' / ', str(max)]
-    __BAR = progressbar.ProgressBar(maxval=max, widgets=widgets)
-    __BAR.start()
-    __BAR.update_interval = 1
-
 
 def update(newval=None):
     """Update the current progress bar.
