diff --git a/ddsmt/progress.py b/ddsmt/progress.py
index c4e1b2b..21654b1 100644
--- a/ddsmt/progress.py
+++ b/ddsmt/progress.py
@@ -18,7 +18,6 @@
 # You should have received a copy of the GNU General Public License
 # along with ddSMT.  If not, see <https://www.gnu.org/licenses/>.
 
-import progressbar
 import sys
 
 __BAR = None
@@ -29,10 +28,6 @@ def start(max):
     global __BAR
     if not sys.stdout.isatty():
         return
-    widgets = [progressbar.Bar(), ' ', progressbar.Counter(), ' / ', str(max)]
-    __BAR = progressbar.ProgressBar(maxval=max, widgets=widgets)
-    __BAR.start()
-    __BAR.update_interval = 1
 
 
 def update(newval=None):
